cranelift/codegen/src/isa/aarch64/inst/emit.rs

```diff
@@ -2756,6 +2756,19 @@ impl MachInstEmit for Inst {
             &Inst::Ret { .. } => {
                 sink.put4(0xd65f03c0);
             }
+            &Inst::AuthenticatedRet { key, is_hint, .. } => {
+                let key = match key {
+                    APIKey::A => 0b0,
+                    APIKey::B => 0b1,
+                };
+
+                if is_hint {
+                    sink.put4(0xd50323bf | key << 6);
```

Comment:
Comment here to note which instruction this is, since it's not the actual `ret`?